drivers/jtag/jtag-aspeed.c

```diff
@@ -248,8 +248,8 @@ static u32 aspeed_jtag_read(struct aspeed_jtag *aspeed_jtag, u32 reg)
 static void aspeed_jtag_write(struct aspeed_jtag *aspeed_jtag, u32 val, u32 reg)
 {
 #ifdef DEBUG_JTAG
```

Comment:
Did we define this? if yes, we should remove it.  In Linux, prints are enabled dynamically using this mechanism - https://www.kernel.org/doc/html/v4.11/admin-guide/dynamic-debug-howto.html

Reply:
No we did not.
Aspeed did in their jtag-aspeed.c file